docs/user/docs/tokens.md

```diff
@@ -41,3 +41,23 @@ authenticate with that against the Collaboration Manager API. One example is:
 ```zsh
 curl -u [username]:[token] https://[baseURL]/api/v1/projects
 ```
+
+Another example is working with the diagram cache of py-capellambse.
+Capellambse is an
+implementation of the capella modelling tool using python and lets you read and
+write models. For more information have a look at the
+[documentation](https://dsd-dbs.github.io/py-capellambse/) or the
+[github repository](https://github.com/DSD-DBS/py-capellambse).
+
+```python
+model = capellambse.model.MelodyModel(
+  path="<path to the model on your machine>",
+  diagram_cache={
+    "path": "http://<your backend url>/api/v1/projects/<your project slug>/models/<your model slug>/diagrams/%s",
```

Comment:
```suggestion
    "path": "https://<your backend url>/api/v1/projects/<your project slug>/models/<your model slug>/diagrams/%s",
```

We should not promote the use of `http`. Instead, we should promote the use of encrypted transfer.